_includes/notes_graph.html

```diff
@@ -46,10 +46,15 @@
     const FONT_BASELINE = 15;
     const MAX_LABEL_LENGTH = 50;
 
-    const graphData = {% include notes_graph.json %}
+    const graphData = {% include tagged_notes_graph.json %}
+    const tag = `{{ include.tag }}`
+    const pageTag = tag.split(" ")[0]
 
-    let nodesData = graphData.nodes;
-    let linksData = graphData.edges;
+    if(tag != "") {
```

Comment:
checking if tags exist on a page in jekyll was proving to be frustrating... so the temporary solution is to abort drawing a graph if a tag is not supplied in JS